x-pack/test/api_integration/apis/ml/indices/field_caps.ts

```diff
@@ -60,17 +60,56 @@ export default ({ getService }: FtrProviderContext) => {
         1,
         `Expected number of indices to be 1, but got ${indices.length}`
       );
-      const fieldsLength = Object.keys(fields).length;
+      const fieldsArr = Object.keys(fields).sort();
 
       // The number of fields returned by the field caps API is different across
       // ES versions in forward compatibility tests. In ES 8.15.0, the `_ignored_source`
       // field was added (https://github.com/elastic/elasticsearch/pull/107567).
       const esVersion = getService('esVersion');
-      const expectedFieldsLength = esVersion.matchRange('>=8.15') ? 22 : 21;
 
-      expect(fieldsLength).to.eql(
-        expectedFieldsLength,
-        `Expected number of fields to be ${expectedFieldsLength}, but got ${fieldsLength}`
+      const expectedFieldsArr = [
+        '@timestamp',
+        '@version',
+        '@version.keyword',
+        '_data_stream_timestamp',
+        '_doc_count',
+        '_feature',
+        '_field_names',
+        '_id',
+        '_ignored',
+        '_index',
+        '_routing',
+        '_seq_no',
+        '_source',
+        '_tier',
+        '_type',
+        '_version',
+        'airline',
+        'responsetime',
+        'sourcetype',
+        'type',
+        'type.keyword',
+      ];
+
+      if (esVersion.matchRange('>=8.15')) {
+        // Remove `_type` field from expected fields array.
+        expectedFieldsArr.splice(expectedFieldsArr.indexOf('_type'), 1);
+        // Add `_ignored_source` and `_nested_path` fields to expected fields arra.
```

Comment:
Typo - `arra`

Reply:
Thanks, fixed in https://github.com/elastic/kibana/pull/188309/commits/c4994d1de52ba891018dbbb7cddbe808bba3732c.